### PR TITLE
CR guide update: 'in QA' label, comment colors

### PR DESCRIPTION
**Pull request by ulaulala**
Jira: -

**Description**
Last decisions about code review improvements are mentioned here:
- using 'in QA' label in your PRs
- using colors in your CR comments to make it more actionable 🚀 

**Self check**

- [x] Self CR'd
- [ ] Tests included
- [ ] Description and screenshots attached

**PR Status**

- [ ] Code Review
- [ ] Quality Assurance

**Blockers**

**Deploy Notes**

### CODE REVIEW

File: src/content/code-review.mdx [hunk 1]
```diff
@@ -97,7 +97,11 @@ Remember about a team goal :)
 ## Comments from reviewers
 
 ### How to be a good reviewer?
-[TBA]
+To make your comments more clear, it will be great to use some colorful icons which will indicate a criticality level.
+- [no color] - something to remember in the future, a question, request for explanation, memes. This comment is not important and doesn't have impact in a review (not a blocker, not an accept).
+- [green] - code changes suggestions, small fixes of namings, interpunction, which doesn't have large impact in code quality. These comments can be skipped by a PR's author. Author can resolve those comments.
```
Comment (mateuszkocz) [inline src/content/code-review.mdx:102]:
```suggestion
- 🟩 - code changes suggestions, small fixes of namings, interpunction, which doesn't have large impact in code quality. These comments can be skipped by a PR's author. Author can resolve those comments.
```

File: src/content/code-review.mdx [hunk 2]
```diff
@@ -97,7 +97,11 @@ Remember about a team goal :)
...
+- [yellow] - potentially important problems, which has an impact in maintenance, tests, or can cause a technical debt. These comments should be covered by a fix, a discussion or a JIRA task with next steps. This comment can be resolved by it's author only. If a 'yellow' comment is added while 'in QA' label is pinned, PR's author can decide about it.
```
Comment (mateuszkocz) [inline src/content/code-review.mdx:103]:
```suggestion
- 🟨 - potentially important problems, which has an impact in maintenance, tests, or can cause a technical debt. These comments should be covered by a fix, a discussion or a JIRA task with next steps. This comment can be resolved by it's author only. If a 'yellow' comment is added while 'in QA' label is pinned, PR's author can decide about it.
```

File: src/content/code-review.mdx [hunk 3]
```diff
@@ -97,7 +97,11 @@ Remember about a team goal :)
...
+- [red] - a code bug, which needs to be fixed. It's an absolute blocker and should be fixed, even if a 'in QA' label is pinned.
```
Comment (mateuszkocz) [inline src/content/code-review.mdx:104]:
```suggestion
- 🟥 - a code bug, which needs to be fixed. It's an absolute blocker and should be fixed, even if a 'in QA' label is pinned.
```

**Review by mateuszkocz — COMMENTED** [review]
Not sure if you still have a problem with those icons so I've added them as suggestions.

**Comment by ulaulala** [comment]
Thanks :)

**Review by mateuszkocz — APPROVED** [review]

**Review by kamilxunai — APPROVED** [review]

**Review by robertgrzonka — APPROVED** [review]
LGTM

**Review by staszewski — APPROVED** [review]

**Review by dkarski — APPROVED** [review]

**Pull request merged**